src/sfr.hpp

```diff
@@ -38,13 +39,15 @@
 
 namespace sfr {
     namespace stabilization {
-        extern float max_time;
-    }
+        // SFRField<float> max_time(1.0f, 0.0f, 2.0f, 1001);
```

Comment:
Normally would req removing commented code but its mine so I'll remove it in the next PR